### ISSUE TITLE
News from date and news to date icons alignment in admin edit product page 

### ISSUE THREAD

**Issue by daniel-ifrim** [magento/magento2#21651]
### Preconditions (*)

1. Magento 2.3.0 with sample data

### Steps to reproduce (*)

1. Go to a product edit page in admin
2. Look on fields 'Set Product as New From' and 'To'.

### Expected result (*)

1. The date picker icon (calendar icon) is not visually aligned. The icon of news from date attribute is over the input text of attribute news to date.

### Actual result (*)

1. The icons should be aligned. The calendar icon should be inside/outside the input fields at the right side. And should not overlap over the next field horizontally. Or something else clean visually.

It seems that the `<button>` tag CSS is missing `position: absolute;` and `display: block;`. I think the CSS  was changed in newer versions. Magento 2.2.6 displays the icons right. Magento 2.2.7, Magento 2.3.0 do not display the icons correctly.

For anyone looking for a quick fix:
```
#app/code/MyNamespace/MyModule/view/adminhtml/web/css/fix-news-from-date.css

.catalog-product-edit input[name="product[news_from_date]"]._has-datepicker + .ui-datepicker-trigger,
.catalog-product-edit input[name="product[news_to_date]"]._has-datepicker + .ui-datepicker-trigger {
    position: absolute;
    display: block;
}
```

```
#app/code/MyNamespace/MyModule/view/adminhtml/layout/catalog_product_edit.xml
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
        <css src="MyNamespace_MyModule::css/fix-news-from-date.css"/>
    </head>
</page>
```
Not much of a great fix but works.

**Comment by magento-engcom-team** [comment]
Hi @daniel-ifrim. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@daniel-ifrim do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by hostep** [comment]
@daniel-ifrim: thanks for the report. I believe this is a duplicate of https://github.com/magento/magento2/issues/18581, which was fixed in https://github.com/magento/magento2/pull/18638 and is to be released in 2.2.8 & 2.3.1 later this month.

Closing as duplicate, if you don't agree, please let me know.
